docs/knowledge_base/tips/the_data_nodes/index.md

```diff
@@ -0,0 +1,151 @@
+  Your browser does not support the video tag.
+</video>
+
+Once you've set up this basic graph, the next step is to create a scenario using it and then submit it for execution.
+
+```python linenums="1"
+scenario = tp.create_scenario(scenario_cfg)
+tp.submit(scenario)
+```
+When submitting the scenario (for execution), Taipy:
+
+- retrieves and reads the model,
+- executes the *predict* function,
+- and writes the results in a Pickle file.
+
+Taipy makes things easy by handling the paths for Pickle files automatically if you haven't defined them. 
+This simplifies the configuration process. When you create several scenarios, 
+the output data nodes from each scenario will automatically point to separate Pickle files.
+
+```python linenums="1"
+scenario_2 = tp.create_scenario(scenario_cfg)
+tp.submit(scenario_2)
+```
+
+In this example, when we create a second scenario, it also brings in a new pair of data nodes: 
+*model* and *predictions*. The *model* data node still points to the same Pickle file because 
+its path was set by the developer in advance. However, the new *predictions* data node points 
+to a different Pickle file. Taipy creates this new Pickle file on the fly during runtime, 
+so it's separate from the one used in the first scenario.
+
+## Tabular Data Nodes
+
+Tabular data nodes in Taipy are a collection of data nodes designed for handling tabular data. 
+By default, the data they point to is presented to the user or developer as a Pandas DataFrame. 
+The predefined tabular data nodes in Taipy include:
+
+- SQL
+- CSV
+- Excel
+- Parquet
+
+These data nodes allow you to work with tabular data from different sources with ease.
+
+<video width="640" height="360" controls>
+  <source src="https://www.taipy.io/wp-content/uploads/2023/03/csv.mp4?_=2" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+To use Tabular data nodes in Taipy, you only need to include them in the configuration 
+and specify certain parameters, like a default path for CSV or *Parquet* files. 
+It's important to note that you can change this path during runtime. 
+For instance, if you create a new scenario, you can instruct the Tabular data nodes 
+to save the results in a different file or directory, which helps you avoid overwriting previous data.
+
+```python linenums="1"
+scenario = tp.create_scenario(scenario_cfg)
+tp.submit(scenario)
+```
+
+When you submit the scenario described above for execution in Taipy, the following steps occur:
+
+- Taipy reads the CSV file named *data.csv* because it is the input data node.
+
+- It takes the data from the CSV file and passes it to the **some_preprocessing** function 
+  using the chosen exposed type, which is typically a Pandas DataFrame by default.
+
+- After the processing is done, Taipy writes or overwrites the result, 
+  which is typically in the form of a Pandas DataFrame, into the Parquet file located at *data.parquet*. 
```

Comment:
What is "\*data.parquet\*" here?

Reply:
Well original content on the web just states: "writes or overwrites the result in the Parquet file located at ‘data.parquet’."

Reply:
I believe @arcanaxion wrote the article. Am I correct?